### ISSUE TITLE
Masks in popups

### ISSUE THREAD

**Issue by estebangrd** [igorescobar/jQuery-Mask-Plugin#364]
I have an input element with a specific class, (i.e. document_mask) within a popup that is created when the popup is opened and I put a mask on it the following way:

$(document).ready(function () {
   var options = getMaskOptions(someParams);
   $('.document_mask').unmask();
   $('.document_mask').mask(options['defaultMask'], options['option']);
}

When option is like:
option['option'] =  {
	onKeyPress: function(val, e, field, option) {
		mask = someMask;
		
		field.mask(mask, option);
	},
	reverse: true
};
option['defaultMask'] = someDefaultMask

Basically I have 2 masks to apply to one input with different document values contained within a popup, i.e. a Modify Selected Element Popup.

The first time the popup is opened, the mask is well applied but when I close it and open another popup which contains another input with a different mask applied on it, it starts switching from the mask from the input contained in the previous (closed and destroyed) popup, to the new mask applied to the new input.

**Comment by igorescobar** [comment]
No working/functinal jsfiddle?
